packages/fhir-converter/src/templates/cda/ValueSet/SystemReference.hbs

```diff
@@ -1,10 +1,16 @@
 {{#if code}}
-    {{#if (eq code '2.16.840.1.113883.6.1')}}
-	    "http://loinc.org"
+    {{#if (startsWith code '1.2.840.114350')}}
+        "https://fhir.epic.com"
```

Comment:
ditto about https, but regarless - would this be fhir? or would this be some hl7 url?

Reply:
thinking about this more, we might just want to remove this. This is only relevant for the identifier field, which will contain a urn:oid to whoever the document came from's internal system. It probably doesn't make sense for us to convert those since there is like an endless number of orgs to support

Reply:
yup agreed

Reply:
removed